slapp/data_selection/README.rst

```diff
@@ -0,0 +1,115 @@
+                  "SELECT oe.id as exp_id, idep.depth, genotypes.name FROM ophys_experiments as oe JOIN ophys_sessions as os on oe.ophys_session_id=os.id JOIN projects on projects.id=os.project_id JOIN imaging_depths as idep on idep.id = oe.imaging_depth_id JOIN specimens on os.specimen_id=specimens.id JOIN donors on specimens.donor_id=donors.id JOIN donors_genotypes dg ON dg.donor_id=donors.id JOIN genotypes ON genotypes.id=dg.genotype_id JOIN genotype_types as gt on gt.id=genotypes.genotype_type_id WHERE gt.name='driver' AND oe.workflow_state = 'passed' AND projects.code='VisualBehaviorTask1B' AND genotypes.name='Sst-IRES-Cre' AND idep.depth BETWEEN 100 and 300",
+                  "SELECT oe.id as exp_id, idep.depth, genotypes.name FROM ophys_experiments as oe JOIN ophys_sessions as os on oe.ophys_session_id=os.id JOIN projects on projects.id=os.project_id JOIN imaging_depths as idep on idep.id = oe.imaging_depth_id JOIN specimens on os.specimen_id=specimens.id JOIN donors on specimens.donor_id=donors.id JOIN donors_genotypes dg ON dg.donor_id=donors.id JOIN genotypes ON genotypes.id=dg.genotype_id JOIN genotype_types as gt on gt.id=genotypes.genotype_type_id WHERE gt.name='driver' AND oe.workflow_state = 'passed' AND projects.code='VisualBehaviorMultiscope' AND genotypes.name='Slc17a7-IRES2-Cre' AND idep.depth BETWEEN 100 and 200",
+                  "SELECT oe.id as exp_id, idep.depth, genotypes.name FROM ophys_experiments as oe JOIN ophys_sessions as os on oe.ophys_session_id=os.id JOIN projects on projects.id=os.project_id JOIN imaging_depths as idep on idep.id = oe.imaging_depth_id JOIN specimens on os.specimen_id=specimens.id JOIN donors on specimens.donor_id=donors.id JOIN donors_genotypes dg ON dg.donor_id=donors.id JOIN genotypes ON genotypes.id=dg.genotype_id JOIN genotype_types as gt on gt.id=genotypes.genotype_type_id WHERE gt.name='driver' AND oe.workflow_state = 'passed' AND projects.code='VisualBehaviorTask1B' AND genotypes.name='Slc17a7-IRES2-Cre' AND idep.depth BETWEEN 100 and 200"],
+          "sub_selection_counts": [75, 25, 20, 10],
+          "random_seed": 42,
+          "comment_string": "example query for README in repo",
+          "log_level": "INFO"
+  }
+   
+running this command will make an entry into a postgres table called ``experiment_selection``. The results can be retrieved by:
```

Comment:
Would probably be helpful at the beginning to link to what kind of database infrastructure is required. I think for LIMS queries we just say LIMS, but we should at least link to the documentation for what databases these functions assume are set up. We could conceivably need to recreate them on a prod server in the future.

Reply:
Added more content at the start. As noted in the the added content, I think future work would be to coherently consolidate docs and design docs into this repo.